### PR TITLE
Celia option 

### PR DESCRIPTION
**Pull request by celiagomezdev**

### CODE REVIEW

**Comment by celiagomezdev** [comment]
Please review this, @casey-collab

File: espanol/ponyrosa/ponyrosa.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Ese pony rosa, que podía hablar, comunicó a los asistentes que el fin andaba cerca, y que todo sería culpa de "Xhsnxbhs" (nombre del actor). 
+Los asistentes, atónitos, abuchearon enérgicamente al pony rosa. -"Nos has hecho un spoiler, ¡tonto!" gritaban, junto a otros improperios.
```
Comment (casey-collab) [inline espanol/ponyrosa/ponyrosa.md]: This line is also too long.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by celiagomezdev** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :smiley_cat:

**Pull request merged**